Added version requirements to vSphere installation preparation doc (#42232)
openshift/openshift-docs: modules/installation-vsphere-infrastructure.adoc

[IMPORTANT]
====
Installing a cluster on VMware vSphere version 6.7U2 or earlier and virtual hardware version 13 is now deprecated. These versions are still fully supported, but support will be removed in version 4.11 of {product-title}. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.

> If your vSphere nodes are below hardware version 15 or your VMware vSphere version is below 6.7U3, you will not be able to upgrade from {product-title} 4.10 to {product-title} 4.11.

Reviewer: ```suggestion
If your vSphere nodes are below hardware version 15 or your VMware vSphere version is below 6.7U3, upgrading from {product-title} 4.10 to {product-title} 4.11 is not available.
```

Author: Made this change

You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
====

[id="vsphere-csi-driver-requirements_{context}"]
= VMware vSphere CSI Driver Operator requirements

The following requirements must be met in order to install the CSI Driver Operator:

* VMware vSphere version 6.7U3 or newer
* Virtual machines of hardware version 15 or newer

> * No third party CSI driver operator already installed in the cluster

Reviewer 2: Here I think we need to add "VMware ESXi version 6.7 Update 3 or greater"

Reviewer 3: I think "VMware vSphere 6.7u3" is fine because it implies both ESXi versions and vcenter versions.
